src/apps/common/submodules/mediaSelect/mediaSelect.js

```diff
@@ -33,8 +38,8 @@ define(function(require) {
 				.append(template);
 
 			callback && callback({
-				getValue: function() {
-					return self.mediaSelectGetValue(template, args);
+				getValue: function(callback) {
+					self.mediaSelectGetValue(template, args, callback);
 				}
```

Comment:
This could be shortened with lodash's help:
```suggestion
				getValue: _.partial(self.mediaSelectGetValue, template, args),
```